dpnp/tests/test_mathematical.py

```diff
@@ -56,18 +59,17 @@ def test_angle_bool(self, deg):
     @pytest.mark.parametrize(
         "dtype", get_all_dtypes(no_bool=True, no_complex=True)
     )
-    @pytest.mark.parametrize("deg", [True, False])
     def test_angle(self, dtype, deg):
         dp_a = dpnp.arange(10, dtype=dtype)
         np_a = dp_a.asnumpy()
 
         expected = numpy.angle(np_a, deg=deg)
         result = dpnp.angle(dp_a, deg=deg)
 
-        assert_dtype_allclose(result, expected)
+        # For dtype=int8, uint8, NumPy returns float16, but dpnp returns float32
+        assert_dtype_allclose(result, expected, check_only_type_kind=True)
```

Comment:
Should we limit kind only check?
```suggestion
        dt_int8 = dtype in [dpnp.int8, dpnp.uint8]
        assert_dtype_allclose(result, expected, check_only_type_kind=dt_int8)
```